src/cli/ops/mount.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * This file is part of clr-boot-manager.
+ *
+ * Copyright © 2016-2018 Intel Corporation
+ * Copyright © 2020 Silke Hofstra
```

Comment:
I don't think you can claim copyright on this file.

Reply:
Actually, this is entirely reasonable.

When you add a new file, or add code that contains substantial amounts of new code authored by your, adding an author/copyright line is very much okay.

Maybe the date should be changed to `2022`, though. Unless it's actually old code that's been circulated before.

Reply:
> Maybe the date should be changed to 2022, though. Unless it's actually old code that's been circulated before.

Looking at my original commit date, this is correct. It has been ready for a while, pending a go-ahead to start integrating `fwupd` for solus.